src/main/resources/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -89,7 +89,7 @@
       "mavenCoordinates": {
          "groupId": "com.google.cloud",
          "artifactId": "google-cloud-video-intelligence",
-         "version": "0.25.0-alpha"
+         "version": "0.25.0-beta"
```

Comment:
Need to fix `"launchStage": "alpha"` above too.

Reply:
done